README.md

```diff
@@ -18,8 +18,8 @@ You can change it inside the script.
 **Input columns**: Discribed in the function *dataFrameNames()* at */TransformingData/transformingData.py* script  
 **Output format**: binary file  
 **Output columns**: same as above + result of the function *featureEngineering()* at */TransformingData/transformingData.py* script    
-**Purpose**: Feature engineering and saving as binary file  
+**Purpose**: Feature engineering - add extra features (listed in return of the *featureEngineering(row)* function in /TransformingData/transformingData.py script) to the original ones (listed in *dataFrameNames()* function in /TransformingData/transformingData.py script). After the transformation new data is saved as binary file (serialization made using the *Pickle* package).  
```

Comment:
There is no information that what is actually pickled are pandas DataFrames.

While I agree that putting exact description of the format in the README will at some point mean that it is outdated, I do not like how it is defined all over the transformingData.py

I would propose to have two globals at the beginning of the script that define both original and new attributes. With a nice comment to easily distinguish which is which.

The README could point to those globals as it currently does. Still it should better describe the format as pandas DataFrames

Reply:
@K4liber Please answer and/or add modification to your code

Reply:
https://github.com/JPETTomography/j-pet-multiphoton-classification/pull/2/commits/c6d4263dfb9ce1c0335812bff7ab0f0074c53b53